### DISCUSSION TITLE
Group by other users' favorite maps (in-game)

### DISCUSSION THREAD

**Discussion by thacer13** [ppy/osu#17669] [Ideas] [answered]
It'd be great to be able to browse and play through favorited maps of any user instantly, if you have those maps installed (it's public info after all). That could work as a dropdown textbox to type the desired username or something.

Right now you could sort of use the internal browser for that, but it's much more cumbersome.

As Spaceman Atlas pointed out, #4919 has to be functional first.

**Comment by bdach** [comment]
https://github.com/ppy/osu/issues/4919

**Reply by thacer13** [reply]
edited

**Comment by peppy** [comment, marked as answer]
I'm so confused. Another player's favourited maps will never be sortable at *song select*. That is a scope beyond going to the moon.

You can view favourites from a user's profile, where you would expect to view them from. You can click any of the favourites on the profile to jump to it in song select. We won't be offering more than this.

**Reply by thacer13** [reply]
fair enough!
